### PR TITLE
doc-update - 9.10.0

### PR DESCRIPTION
**Pull request by Saksham-Nautiyal**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,32 @@ Dell OpenManage Ansible Modules Release Notes
 
 .. contents:: Topics
 
+v9.10.0
+=======
+
+Release Summary
+---------------
+
+- Modules are added to support OpenManage Enterprise Integration for VMWare vCenter Plug-in. - The ``omevv_firmware`` module is added to support the firmware update on the host and single cluster. - The ``omevv_firmware_repository_profile`` module is enhanced to support resync of repository profiles from the UMP plugin. - The ``idrac_certificates`` module is enhanced to support SSL CSR generation for 4096 key size. (Issue #737)
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- The ``omevv_firmware`` module is added to support the firmware update of the single host and single cluster.
- The ``omevv_firmware_repository_profile`` module is enhanced to support resync of repository profiles from the UMP plugin.
- The ``idrac_certificates`` module is enhanced to support SSL CSR generation for 4096 key size. (Issue #737) 
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,6 +4,32 @@ Dell OpenManage Ansible Modules Release Notes
...
+Release Summary
+---------------
+
+- Modules are added to support OpenManage Enterprise Integration for VMWare vCenter Plug-in. - The ``omevv_firmware`` module is added to support the firmware update on the host and single cluster. - The ``omevv_firmware_repository_profile`` module is enhanced to support resync of repository profiles from the UMP plugin. - The ``idrac_certificates`` module is enhanced to support SSL CSR generation for 4096 key size. (Issue #737) 
+
+Major Changes
+-------------
+
+- omevv_firmware - This module allows to update firmware on the host and single cluster.
+- omevv_firmware_repository_profile - This module allows to resync the repository profiles from the UMP plugin.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]: Add this under minor changes section and also include "idrac_certificate" point

File: changelogs/changelog.yaml [hunk 3]
```diff
@@ -2177,3 +2177,46 @@ releases:
...
+          firmware update using HTTP, HTTPS, and FTP shares with
+          authentication on iDRAC8.
+        - ome_smart_fabric_uplink - Issue(186024) - The module supported by
+          OpenManage Enterprise Modular, however it does not allow the
+          creation of multiple uplinks of the same name. If an uplink is
+          created using the same name as an existing uplink, then the existing
+          uplink is modified.
+      modules:
+        - description:
+            This module allows to update firmware on the host and
```
Comment (rajshekarp87) [inline changelogs/changelog.yaml]:
```suggestion
            This module allows to update firmware of the single host and
```

File: changelogs/changelog.yaml [hunk 4]
```diff
@@ -2177,3 +2177,46 @@ releases:
           name: omevv_baseline_profile
           namespace: ''
     release_date: '2024-11-30'
+
+  9.10.0:
+    changes:
+      release_summary: '- Modules are added to support OpenManage Enterprise
```
Comment (rajshekarp87) [inline changelogs/changelog.yaml]:
```suggestion
      release_summary: '
```

File: changelogs/changelog.yaml [hunk 5]
```diff
@@ -2177,3 +2177,46 @@ releases:
...
+          Integration for VMWare vCenter Plug-in.
```
Comment (rajshekarp87) [inline changelogs/changelog.yaml]:
```suggestion

```

File: changelogs/changelog.yaml [hunk 6]
```diff
@@ -2177,3 +2177,46 @@ releases:
...
+        - The ``omevv_firmware`` module is added to support the firmware update
+          on the host and single cluster.
```
Comment (rajshekarp87) [inline changelogs/changelog.yaml]:
```suggestion
          of the single host and single cluster.
```

File: changelogs/changelog.yaml [hunk 7]
```diff
@@ -2177,3 +2177,46 @@ releases:
...
+        - The ``omevv_firmware_repository_profile`` module is enhanced to
+          support resync of repository profiles from the UMP plugin.
```
Comment (rajshekarp87) [inline changelogs/changelog.yaml]: elaborate UMP

File: CHANGELOG.rst [hunk 8]
```diff
@@ -4,6 +4,32 @@ Dell OpenManage Ansible Modules Release Notes
...
+
+Release Summary
+---------------
+
+- Modules are added to support OpenManage Enterprise Integration for VMWare vCenter Plug-in. - The ``omevv_firmware`` module is added to support the firmware update on the host and single cluster. - The ``omevv_firmware_repository_profile`` module is enhanced to support resync of repository profiles from the UMP plugin. - The ``idrac_certificates`` module is enhanced to support SSL CSR generation for 4096 key size. (Issue #737) 
+
+Major Changes
+-------------
+
+- omevv_firmware - This module allows to update firmware on the host and single cluster.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- omevv_firmware - This module allows to update firmware of the single host and single cluster.
```

File: CHANGELOG.rst (hunk 2 above)
Comment (Saksham-Nautiyal) [reply CHANGELOG.rst]: certificate is bug fix

File: CHANGELOG.rst [hunk 9]
```diff
@@ -32,7 +67,11 @@ v9.8.0
 Release Summary
 ---------------
 
-- Modules are added to support OpenManage Enterprise Integration for VMWare vCenter Plug-in. - The ``omevv_vcenter_info`` module is added to support retrieval of vCenter information. - The ``omevv_firmware_repository_profile_info`` module is added to support retrieval of firmware repository profile information. - The ``omevv_firmware_repository_profile`` module is added to support management of firmware repository profile. - The ``idrac_firmware_info`` module is enhanced to support iDRAC10 and OMSDK dependency is removed.
+- Modules are added to support OpenManage Enterprise Integration for VMWare vCenter Plug-in.
```
Comment (rajshekarp87) [inline CHANGELOG.rst:70]:
```suggestion

```

Comment (Saksham-Nautiyal) [reply CHANGELOG.rst:70]: done

**Review by rajshekarp87 — DISMISSED** [review]

**Review by ABHISHEK-SINHA10 — DISMISSED** [review]

**Comment by ABHISHEK-SINHA10** [comment]
@Saksham-Nautiyal  there's no README changes this time ? please check once

**Review by ABHISHEK-SINHA10 — APPROVED** [review]

**Review by rajshekarp87 — APPROVED** [review]

**Pull request merged**